swap/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 description = "XMR/BTC trustless atomic swaps."
 
 [[bin]]
-name = "swap_cli"
+name = "buy_xmr"
```

Comment:
I think this section is actually redundant? We are not specifying the asb here and things still work so I think you can just delete that.

Reply:
yeh it is. I have also forgotten to change all the CI mentions of swap_cli